test/integration/targets/dnf/tasks/main.yml

```diff
@@ -62,5 +62,5 @@
         (ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_major_version is version('8', '>='))
 
 - include_tasks: nobest.yml
-  when: (ansible_distribution == 'Fedora' and ansible_distribution_major_version is version('24', '>=') and ansible_distribution_major_version is version('31', '<=')) or
+  when: (ansible_distribution == 'Fedora' and ansible_distribution_major_version is version('24', '>=') and ansible_distribution_major_version is version('32', '<=')) or
```

Comment:
I wonder why the `ansible_distribution_major_version is version('32', '<='))` conditional is even there? Is the `nobest` functionality not going to work/be available in Fedora 33+?

Reply:
Maybe, Fedora Team may have a delay on preparing docker repo on Fedora 33 like Fedora 32.

Reply:
Are we going to keep incrementing this each time Fedora X is released and docker repo for Fedora X is available?

Reply:
Maybe if I could change `nobest` integration test later and it doesn't depend on `docker` anymore, that would be better.

For now I don't have any idea.

Reply:
We should remove this conditional altogether. The versions of Fedora we test with in CI should all support this version. If it's not working, we will see it when we are adding new Fedora versions.

Reply:
@samdoran, OK, I will remove the condition, so it will be tested on all Fedora versions greater than 24.

Reply:
Fixed in https://github.com/ansible/ansible/pull/71805/commits/f2bcba283a84d1349c43dc3708d6a9db4e0412fb.

@mkrizek Is it ok now?